bower.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "jQuery": "~2.2.2",
     "bootstrap": "~3.3.6",
-    "angular": "~1.5.3"
+    "angular": "~1.5.3",
+    "angular-route": "~1.5.3"
```

Comment:
We have, and I like it.  It's got more power.  But I think 22-23 used ngRoute so there should be some material, right?  
I don't know if there's any benefit to teaching one vs. the other since students rarely make much use of it.  
It might be tough to re-write it to use another router.